src/main/java/ch/njol/skript/expressions/ExprMaxHealth.java

```diff
@@ -66,13 +66,13 @@ protected String getPropertyName() {
 		return "max health";
 	}
 	
-	@Override
-	@Nullable
-	public Class<?>[] acceptChange(final ChangeMode mode) {
-		if(!Skript.isRunningMinecraft(1, 5, 2)) {
-			Skript.error("The max health of an entity can only be changed in Minecraft 1.6 and later");
-			return null;
-		}
+	@Override
+	@Nullable
+	public Class<?>[] acceptChange(final ChangeMode mode) {
+		if(!Skript.isRunningMinecraft(1, 5, 2)) {
+			Skript.error("The max health of an entity can only be changed in Minecraft 1.6 and later");
+			return null;
+		}

```

Comment:
What changed here, the line endings? Hmmmm
*Those four lines could be removed altogether btw, probably out of scope though, but if they are already changed..*